packages/core/src/scripts/migrate-oz-cli-project.ts

```diff
@@ -0,0 +1,426 @@
+import path from 'path';
+import chalk from 'chalk';
+import { promises as fs } from 'fs';
+import { ManifestData, ImplDeployment } from '../manifest';
+import type { StorageItem, StorageLayout, TypeItem, TypeMembers, StructMember } from '../storage';
+
+const OPEN_ZEPPELIN_FOLDER = '.openzeppelin';
+const EXPORT_FILE = 'openzeppelin-cli-export.json';
+const PROJECT_FILE = path.join(OPEN_ZEPPELIN_FOLDER, 'project.json');
+const SUCESS_CHECK = chalk.keyword('green')('✔') + ' ';
+
+export async function migrateLegacyProject(): Promise<void> {
+  const manifestsMigrationData = await migrateManifestFiles();
```

Comment:
Can you rename the variable as well as the property in `MigrationOutput` to something containing "export data"? Perhaps `networkExportData`. I'm still confused by the term "migration data".